src/libsync/resolveurljobfactory.cpp

```diff
@@ -0,0 +1,56 @@
+#include "resolveurljobfactory.h"
+#include "common/utility.h"
+#include <QNetworkReply>
+
+namespace {
+Q_LOGGING_CATEGORY(lcResolveUrl, "wizard.resolveurl")
+}
+
+namespace OCC {
+
+ResolveUrlJobFactory::ResolveUrlJobFactory(QNetworkAccessManager *nam, QObject *parent)
```

Comment:
Only relevant in the new wizard, no need to add it to libsync?
and unrelated to the pr?